src/sota_tools/ostree_hash.cc

```diff
@@ -15,7 +15,7 @@ OSTreeHash OSTreeHash::Parse(const std::string& hash) {
   for (int i = 0; i < 32; i++) {
     char byte_string[3];
     byte_string[2] = 0;
-    unsigned long byte_holder;
+    uint64_t byte_holder;
```

Comment:
There is some major irony in using a `uint64_t` to store a byte.

Reply:
)))